webpack.config.js

```diff
@@ -64,15 +65,7 @@ const webpackConfig = {
 			}
 		]
 	},
-	resolve: {
```

Comment:
Hi @aduth, the main reason I had separated the `resolve` into its own separate file was due to the fact that `webpack.config.js` has `lib/create-config` in its import path. 

Easy repro: `node -e "var config = require('./webpack.config');"` (execute this at project root)

There doesn't seem to be an easy way to modify the node path for ESLint - what do you suggest?